shaders/src/getters.fs

```diff
@@ -65,6 +65,14 @@ float getNdotV() {
     return shading_NoV;
 }
 
+mat3 getTriplanarOrientationMatrix() {
+    return objectUniforms.triplanarOrientationMatrix;
+}
+
+vec3 getObjectAabbCenterWorldSpace() {
+    return objectUniforms.worldAabbCenter;
+}
```

Comment:
Is [GFX-2583](https://shapr3d.atlassian.net/browse/GFX-2583) really completed in this PR? I mean the shaders aren't updated yet.

Reply:
Edited the description to include a link to the Shapr-side PR: https://github.com/shapr3d/shapr3d/pull/13806

Reply:
Fixed in 86c38a3